packages/flutter_driver/lib/src/driver/timeline_summary.dart

```diff
@@ -5,6 +5,7 @@
 import 'dart:convert' show JsonEncoder, json;
 import 'dart:math' as math;
 
+import 'package:collection/collection.dart';
```

Comment:
flutter_driver does not have a "real" direct dependency on collection, so we shouldn't use it here. The only reason it ends up in the pubspec is because of an indirect dependency that we automate into the pubspec - you can tell because of the `THIS LINE IS AUTOGENERATED` comment next to it. We could bump this to a real dependency, but generally we avoid this package in this repository to avoid users having weird constraints for downstream consumers

Reply:
Oh sure! I learnt this lesson in another PR (but this PR is submitted before I learnt this)